client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java

```diff
@@ -145,11 +190,9 @@ private static String handlePreparedStatement(PreparedStatement stmt) {
             sql = extractSqlFromH2PreparedStatement(stmt);
         }
 
-        /*
-            handle db middleware, ie, zebra
-         */
+        // just check whether there exist any new update
         if (fullClassName.startsWith("com.dianping.zebra")){
-            sql = extractSqlFromZebraPreparedStatement(stmt);
+            throw new IllegalArgumentException("unsupported type for zebra: " + fullClassName);
         }
```

Comment:
why throwing an exception here?  wasn't this checked with `checkZebraPreparedStatementWrapper`?  or should rather be an `assert` here?

Reply:
all zebra prepared statements should be handled before this line (see 155).
if this line is reached, it means that there exist new types of prepared statements, then throw the exception with the info.

Reply:
ok. can u add this clarification as a code comment?

Reply:
the comment is modified with more info now. thanks.